python/paddle/v2/fluid/layers/nn.py

```diff
@@ -25,32 +25,48 @@ def fc(input,
        act=None,
        name=None):
     """
-    Fully Connected Layer.
+    **Fully Connected Layer**
+
+    This layer accepts multiple inputs and applies a linear transformation to each input.
+    If activation type is provided, the corresponding non-linear function is applied to the
```

Comment:
minor: Did you mean "activation" instead of "non-linear" in 'the corresponding non-linear function'?

Reply:
Fixed in https://github.com/PaddlePaddle/Paddle/pull/6806/commits/5d3131cdc7edbf53ddb85935940f90ec51e65e35